vsm

```diff
@@ -164,8 +193,12 @@ def run(state):
         else:
             process(state, line, None)
 
+def get_runtime():
+    return round(time.time() * 1000 - start_time_ms)
```

Comment:
Good idea. By the way, I do fill in `start_time_ms` at the top of the `__init__()` method, so that initial assignment to `start_time_ms` is just to declare the variable globally.